frontend/src/scenes/trends/trendsLogic.ts

```diff
@@ -346,12 +346,13 @@ export const trendsLogic = kea<trendsLogicType<IndexedTrendResult, TrendResponse
             actions.loadResults()
         },
         loadResultsSuccess: () => {
-            if (!props.dashboardItemId) {
+            if (!insightLogic.values.insight.id) {
                 insightHistoryLogic.actions.createInsight({
                     ...values.filters,
                     insight: values.filters.session ? ViewType.SESSIONS : values.filters.insight,
                 })
             }
+            insightLogic.actions.updateInsightFilters(values.filters)
```

Comment:
I think this is also started when we're on a dashboard, so it's anybody's guess what value `insightLogic` will be pointing to.